langroid/utils/configuration.py

```diff
@@ -22,7 +22,7 @@ class Config:
         extra = "forbid"
 
 
-load_dotenv()  # get settings from .env file
+load_dotenv(find_dotenv(usecwd=True))  # get settings from .env file
```

Comment:
plain load_dotenv() occurs in several more places. Do you recommend doing this everywhere?

Reply:
Yeah, at least anywhere in the main langroid source (since it shouldn't search based on the location where langroid is installed). Leaving plain load_dotenv() should be fine in example code and similar.